[2022-08-12 03:55:46,996] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-08-12 03:55:47,009] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-08-12 03:55:47,009] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 03:55:47,010] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-12 03:55:47,010] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 03:55:47,030] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-11 00:00:00+00:00
[2022-08-12 03:55:47,048] {standard_task_runner.py:52} INFO - Started process 26338 to run task
[2022-08-12 03:55:47,056] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-11T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpr00cbgud', '--error-file', '/tmp/tmpgzas9vos']
[2022-08-12 03:55:47,059] {standard_task_runner.py:80} INFO - Job 169: Subtask link_dag
[2022-08-12 03:55:47,219] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-11T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-12 03:55:47,396] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-11T00:00:00+00:00
[2022-08-12 03:56:33,816] {python.py:173} INFO - Done. Returned value was: None
[2022-08-12 03:56:33,833] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220811T000000, start_date=20220812T035546, end_date=20220812T035633
[2022-08-12 03:56:33,924] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-12 03:56:33,961] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
